src/org/opensolaris/opengrok/index/IgnoredNames.java

```diff
@@ -31,10 +32,10 @@
  *
  * @author Vladimir Kotal
  */
-public class IgnoredNames {
+public class IgnoredNames implements Serializable {
```

Comment:
Shouldn't serializable object have some kind of `long` ID? Like [here](https://github.com/OpenGrok/OpenGrok/blob/master/src/org/opensolaris/opengrok/history/GitRepository.java#L58) in `GitRepository`.